src/components/event/MapModal/Map-modal.js

```diff
@@ -37,9 +39,7 @@ export const MapModal = ({ initialize, values }) => {
       selectedPos: { latitude: null, longitude: null },
     });
 
-    return values.selectedPos.latitude
-      ? startValue
-      : startValue && setOpen(false);
+    return latitude ? startValue : startValue && setOpen(false);
```

Comment:
Why condition on `startValue` placed after `:` symbol. Please move it on start: `latitude && startValue ? .. : ..`